examples/packet.arm64.yml

```diff
@@ -1,3 +1,9 @@
+kernel:
+  image: linuxkit/kernel:4.9.76
+  cmdline: "console=ttyAMA0"
+  # Note: ucode is only set to "" to override the amd64 setting
+  # Can be left out for arm64 builds.
```

Comment:
This file could perhaps benefit from a little context, i.e. a comment at the top which says something like:
```
# This file can be built together with the x86_64 specific settings in packet.yml to produce an image suitable for arm64. See docs/platform-packet.md for detail.
```
This then gives a bit more context for the comment above. I think I'd also make the second line more explicit, something like "If writing a yml specifically for arm64 then this can be left out".